aiogram/utils/keyboard.py

```diff
@@ -152,7 +152,7 @@ def export(self) -> List[List[ButtonType]]:
 
         .. code-block:: python
 
-            >>> builder = KeyboardBuilder(button_type=InlineKeyboardButton)
+            >>> builder = InlineKeyboardBuilder(button_type=InlineKeyboardButton)
```

Comment:
Since this line of documentation is shown in all child classes, this change does not make this example any less confusing for all other cases, so I disagree with this change.

- https://docs.aiogram.dev/en/stable/utils/keyboard.html#aiogram.utils.keyboard.InlineKeyboardBuilder.export
- https://docs.aiogram.dev/en/stable/utils/keyboard.html#aiogram.utils.keyboard.ReplyKeyboardBuilder.export

Let's find another way.